src/Microsoft.AspNet.Mvc.Core/ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -334,14 +335,24 @@ public override bool IsBindingRequired
         {
             get
             {
-                if (MetadataKind == ModelMetadataKind.Property)
-                {
-                    return BindingMetadata.IsBindingRequired;
-                }
-                else
+                if (!_isBindingRequired.HasValue)
                 {
-                    return false;
+                    if (MetadataKind == ModelMetadataKind.Type)
+                    {
+                        _isBindingRequired = false;
```

Comment:
Would would that even mean semantically? We have a few MM properties like this and aren't meaningful for a type.